### PR TITLE
Allow to initialise New Relic app_name

### PR DESCRIPTION
**Pull request by dquerioux**
Add app_name option for New Relic handler.

### CODE REVIEW

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -292,6 +293,7 @@ public function getConfigTreeBuilder()
                             ->scalarNode('priority')->defaultValue(0)->end()
                             ->scalarNode('level')->defaultValue('DEBUG')->end()
                             ->booleanNode('bubble')->defaultTrue()->end()
+                            ->scalarNode('app_name')->defaultValue(null)->end()
```
Comment (xabbuh) [inline DependencyInjection/Configuration.php]: You need to update the XSD file too.

Comment (dquerioux) [inline DependencyInjection/Configuration.php]: You are right.
It's done?

Thanks

File: Resources/config/schema/monolog-1.0.xsd [hunk 2]
```diff
@@ -32,6 +32,7 @@
         <xsd:attribute name="priority" type="xsd:integer" />
         <xsd:attribute name="level" type="level" />
         <xsd:attribute name="bubble" type="xsd:boolean" />
+        <xsd:attribute name="app_name" type="xsd:string" />
```
Comment (xabbuh) [inline Resources/config/schema/monolog-1.0.xsd]: should be `app-name`

File: DependencyInjection/Configuration.php (hunk 1 above)
Comment (stof) [inline DependencyInjection/Configuration.php]: Please use ``defaultNull()`` here for consistency

**Comment by stof** [comment]
@dquerioux please update the PR to make it ready to be merged (I would like to do a release this evening)

**Comment by dquerioux** [comment]
Thanks for the review xabbuh,
I updated the PR with your recomandations.

Is it good for you stof?

**Pull request merged**